mantidproject/mantid
# Add improvement of minimizer documentation to release notes

Signed-off-by: Karl Palmen <karl.palmen@stfc.ac.uk>

Description of work.

For release 3.9 improvements were made to the fitting minimizer documentation (in #15014) and 
this needed mentioning in the release notes.

**To test:**

Build the documentation and check that the improvement of the fitting function minimizers has 
been appropriately added to the release notes of release 3.9.

Fixes #15609.



---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
